meetings/2024/notes-2024-04-29.md

```diff
@@ -131,6 +131,22 @@ RGN: True, but in the case of syntax errors, you –
 
 EAO: Yes, syntax errors are different.
 
-TIM: This means that #738 will be blocked on an issue that MIH will raise.
+RGN: Yes, I agree. The other JS Intl APIs provide a fallback value or best effort return value when formatting. But it makes sense to me that MessageFormat is different enough that it would be useful to return the error in addition to returning a value.
+
+ECH: I'd like to understand RGN's reasoning on what makes MessageFormat different from the other JS Intl APIs in his mind, that MessageFormat should return the error(s) and a return value when the other APIs don't.
+
+RGN: I think it's the impact of MessageFormat. MessageFormat will in turn call the other APIs, so the scope of the API is bigger, and the ways that the API to go wrong is greater.
+
+EAO: In addition, because MessageFormat's formatting output is larger, since it returns full messages that may in turn contain the output of the other APIs. So when there is an error in formatting Message Format, the difference between the developer expectation and what the user sees is greater.
```

Comment:
That's not quite what I said.

```suggestion
EAO: In addition, because MessageFormat depends on user data, the distance between the developer and the user's localized experience is much greater. If an error is introduced during translation, it might not even be possible to catch that in CI systems or elsewhere, before it goes to production.
```